key_files/amoc_26n.yml

```diff
@@ -0,0 +1,15 @@
+---
+#AMOC_26N
+name: AMOC_26N
+units: Sv
+dimensions: 1
+model: NEMO
+modelFiles: $BASEDIR_MODEL/$JOBID/nemo_$JOBIDo_1y_*_grid-V.nc
```

Comment:
you aware that yaml file parsing will not detect these as system/environment variables - they're mere strings for pyyaml - do you convert them to sys/env variables inside the code?

Reply:
Yeah, we have a tool to replace them when they're loaded.